.github/workflows/ci-build.yml

```diff
@@ -57,6 +57,53 @@ jobs:
           echo $'Examples\n========' > doc/source/examples/index.rst
           make -C doc html SPHINXOPTS="-j auto -w build_errors.txt -N"
 
+      - name: Build json documentation
+        run: |
+          # Export the documentation files to JSON files.
+          sphinx-build -M json doc/source doc-json -j auto -w build_errors.txt -N;
+
+      - name: Flatten the generated nested files into a single directory
+        run: |
+          echo Flattening a nested directory
+          mkdir doc-flatten-json;
+          echo Move all the JSON file to the flatten directory.
+          find . -name "*.fjson" -exec mv "{}" --backup=numbered ./doc-flatten-json \;
+          echo Make sure all the file has .json extensions instead of the .fjson
+          for f in doc-flatten-json/* ; do mv -- "$f" "$f.json" ; done;
```

Comment:
if ``f`` is the complete file name, you might end up with ``file.json.json`` or ``file.doc.json``.

Reply:
![image](https://user-images.githubusercontent.com/28149841/193007009-959cc332-dbbc-48c4-bf60-2d0153ff3393.png)

Reply:
Suggested solution:

https://stackoverflow.com/a/71969517/6650211

Reply:
Improved:
![image](https://user-images.githubusercontent.com/87315832/195416032-addac88d-09c6-4f88-9991-97a24df2ae24.png)

I will check if the backed up files are needed.